app/cafe.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+
+from app.errors import NotVaccinatedError
+from app.errors import OutdatedVaccineError
+from app.errors import NotWearingMaskError
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        vaccine = visitor.get("vaccine", {})
+        expiration_date = vaccine.get("expiration_date")
+
+        if not vaccine:
+            raise NotVaccinatedError(
+                "All friends should be vaccinated")
+        if (not expiration_date
+                or expiration_date < datetime.date.today()):
+            raise OutdatedVaccineError(
+                "All friends should be vaccinated")
+        if not visitor.get("wearing_a_mask", False):
+            raise NotWearingMaskError(
+                "Friends should buy {masks_to_buy} masks")
```

Comment:
The error message for `NotWearingMaskError` should dynamically include the number of masks to buy. Currently, it uses a placeholder `{masks_to_buy}` which is not defined or replaced with an actual value. Consider revising this to ensure the message is descriptive and accurate as per the checklist requirement: 'Use descriptive error messages'.